vulcanai2/models/dnn.py

```diff
@@ -157,8 +157,8 @@ def _create_network(self, **kwargs):
         self.network = nn.Sequential(dense_layers)
 
         if self._num_classes:
-            if self._get_out_dim is not None and \
-                len(self._get_out_dim()) >= 2:
+            if self.input_networks is not None and\
```

Comment:
Actually I don't think it is needed to add flatten since it comes after a dense network always. the input networks flow through the current network first and thus won't need to be flattened